star/private/tdc_hydro.f90

```diff
@@ -46,6 +46,15 @@ subroutine set_viscosity_vars_TDC(s, ierr)
       ierr = 0
       op_err = 0
```

Comment:
Do we actually use `op_err`? It looks like both it and `ierr` and initialised as zero, then `ierr` is changed to match `op_err` whenever the latter is non-zero. But that seems the same as just using `ierr` instead of `op_err`.